### PR TITLE
pancan meta study fix

### PR DESCRIPTION
**Pull request by yichaoS**
Fixing: https://github.com/cBioPortal/datahub/issues/470 and  https://github.com/cBioPortal/datahub/issues/168 (pancan studies only)

### CODE REVIEW

File: public/acc_tcga_pan_can_atlas_2018/meta_study.txt [hunk 1]
```diff
@@ -3,4 +3,5 @@ cancer_study_identifier: acc_tcga_pan_can_atlas_2018
 name: Adrenocortical Carcinoma (TCGA, PanCancer Atlas)
 short_name: ACC (TCGA PanCan)
 description: Adrenocortical Carcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
-groups: PUBLIC;PANCAN
\ No newline at end of file
+groups: PUBLIC;PANCAN
+pmid: 29625048, 29596782, 29622463, 29617662, 29625055, 29625050
```
Comment (jjgao) [inline public/acc_tcga_pan_can_atlas_2018/meta_study.txt]: I think we also need a line `citation: TCGA, Cell 2018`

**Comment by yichaoS** [comment]
@jjgao Added citation. Not just waiting for @sandertan to update the validator. Then I'll ask @sheridancbio 's help to merge the change in DB and migrate (and update importer).

File: public/acc_tcga_pan_can_atlas_2018/meta_study.txt [hunk 2]
```diff
@@ -4,4 +4,5 @@ name: Adrenocortical Carcinoma (TCGA, PanCancer Atlas)
 short_name: ACC (TCGA PanCan)
 description: Adrenocortical Carcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
 groups: PUBLIC;PANCAN
-pmid: 29625048, 29596782, 29622463, 29617662, 29625055, 29625050
\ No newline at end of file
+pmid: 29625048, 29596782, 29622463, 29617662, 29625055, 29625050
```
Comment (jjgao) [inline public/acc_tcga_pan_can_atlas_2018/meta_study.txt]: please remove space.

Comment (yichaoS) [reply public/acc_tcga_pan_can_atlas_2018/meta_study.txt]: just removed @jjgao

File: public/acc_tcga_pan_can_atlas_2018/meta_study.txt (hunk 1 above)
Comment (yichaoS) [reply public/acc_tcga_pan_can_atlas_2018/meta_study.txt]: @jjgao just updated.

**Comment by inodb** [comment]
Merging this by override as requested by @yichaoS (validator is not up to date)

**Comment by inodb** [comment]
Thanks @sandertan ! Sorry, should've paid better attention. @yichaoS could u maybe elaborate why the merge was necessary so we can figure out how to improve the process?

My understanding is: currently to get things imported in public portal it needs to be in datahub. The backend team already applied those changes of the PR to the database, so the import of this study would work fine. Just the validator would error. Is that correct? Thanks!

**Comment by yichaoS** [comment]
@sandertan @ino Yes, before this merge, the database migration's done and the imported's updated (@sheridancbio) so the importing has been going smoothly. The publication link has been broken for all pancan studies on public portal so we would like to address this asap. 

@sandertan @inodb @sheridancbio As Ino mentioned, maybe it's helpful to define/streamline the process of updating different components revolved around DB schema change.

**Comment by sheridancbio** [comment]
Yes, I think we knew that the validation tests would fail until the master branch received the updated validation test code. In circumstances where the database schema is in the process of modification (in rc / release branch) and has been partially deployed into production, we should define what is the proper way to import data into the (ahead of master) schema database. Perhaps a prospective study data subdirectory/repository is needed, and then data migrates to the production area when the schema changes become standard (master)? It is worth thinking about options.

**Pull request merged**